syntax = "proto3";

package robot;

message Robot {
    repeated double joint_angles = 1;
    repeated double joint_velocities = 2;
    repeated double tcp_pose = 3;
    repeated double tcp_velocity = 4;
    repeated double tcp_force = 5;
}